packages/base-controller/src/RestrictedMessenger.test.ts

```diff
@@ -1120,4 +1120,211 @@ describe('RestrictedMessenger', () => {
     expect(pings).toBe(1);
     expect(currentCount).toBe(10);
   });
+
+  describe('registerMethodActionHandlers', () => {
+    it('should register action handlers for specified methods on the given restricted messaging client', () => {
```

Comment:
The messenger is restricted, but the messenger client is not (it's just an object).

``` suggestion
    it('should register action handlers for specified methods on the given messenger client', () => {